Java/testutils/src/main/java/com/amazon/randomcutforest/testutils/NormalMixtureTestData.java

```diff
@@ -47,6 +47,10 @@ public NormalMixtureTestData() {
         this(0.0, 1.0, 4.0, 2.0, 0.01, 0.3);
     }
 
+    public NormalMixtureTestData(double baseMu) {
+        this(baseMu, 1.0, 4.0, 2.0, 0.01, 0.3);
```

Comment:
It seems a little weird to give the option to only change one of the mean values in the mixture distribution.

Reply:
True :) Will update.